poleno/utils/tests/test_templatetags.py

```diff
@@ -253,3 +256,9 @@ def test_change_lang_tag(self):
             self.assertEqual(r1.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r2.content, u'(/en/language/)(/de/language/)(/fr/language/)')
             self.assertEqual(r3.content, u'(/en/language/)(/de/language/)(/fr/language/)')
+
+        patched = functools.partial(page_not_found, template_name=u'404x.html')
```

Comment:
@martinmacko47 Je toto adekvatna nahrada `404.html` alebo si si to predstavoval inak?

Reply:
@viliambalaz Nie je to dobre. Tento test ma predsa kontrolovat, ze fix `change_lang` template tagu spraveny v PR #304 funguje. Inymi slovami, ked docasne zvratis spat fix `change_lang` z PR #304, tak tento test by mal padnut. Ale nepadne. Vyskusaj si to.

Tento test musi testovat zavolatnie `change_lang` pri vyhodnocovani requestu s url bez `/en` prefixu. To sa vsak nestane.

Ako som pisal v https://github.com/martinmacko47/chcemvediet/pull/304#discussion_r502814617 treba:

> Jedine, co mi napada je spravit si vlastny 404 template, kde sa pouzuje `change_lang`. A teda bez fixu `self.client.get(u'/language/')` padne na errore v tem 404 template.

Cize namiesto overridnutia 404 templatu neexistujucim templatom, treba spravit vlastny 404 template, ktory v sebe pouzije `change_lang`. A teda bez fixu z PR #304 padne.

Reply:
:ok: